apps/exports/views.py

```diff
@@ -27,12 +28,14 @@ def home(request):
 
 @login_required
 def create_export_config(request):
+
     if request.method == 'POST':
         form = ExportConfigForm(request.POST, request.FILES)
         if form.is_valid():
-            export = form.save(commit=False)
-            export.created_by = request.user
-            export.save()
+            with reversion.create_revision():
+                export = form.save(commit=False)
+                export.created_by = request.user
+                export.save()
```

Comment:
could we overwrite the `save` call to always create a revision so callers don't have to remember to do that? or is that not best practice for some reason?

Reply:
nah I think it should be fine without violating any good practice